packages/sdks-tests/src/e2e-tests/editing.spec.ts

```diff
@@ -150,6 +150,33 @@ test.describe('Visual Editing', () => {
     }
   });
 
+  test('removal of styles should work properly', async ({ page, packageName, sdk, basePort }) => {
```

Comment:
ran the test on main branch as well and it was failing so we can be sure that styles now work as expected,

![Screenshot 2025-02-18 at 7 37 48 PM](https://github.com/user-attachments/assets/c6c75a91-0438-466a-b01d-3924a7f47321)